guide/anatomy.rst

```diff
@@ -223,29 +223,26 @@ below, taken from one of the samples in our github repository::
             self.right_motor = wpilib.Spark(1)
             self.drive = wpilib.drive.DifferentialDrive(self.left_motor, self.right_motor)
             self.stick = wpilib.Joystick(1)
-    
+            self.timer = wpilib.Timer()
+
         def autonomousInit(self):
             """This function is run once each time the robot enters autonomous mode."""
-            self.auto_loop_counter = 0
-    
+            self.timer.reset()
+            self.timer.start()
+
         def autonomousPeriodic(self):
             """This function is called periodically during autonomous."""
-    
-            # Check if we've completed 100 loops (approximately 2 seconds)
-            if self.auto_loop_counter < 100:
+
+            # Drive for two seconds
+            if self.timer.get() < 2.0:
```

Comment:
I like using `timer.hasPeriodPassed` because that'll reset it so you can chain these. Otherwise you gotta do quick math

Reply:
I'd prefer to keep it as-is to match upstream, but it can be changed if others agree.

Reply:
Oh is this supposed to match the screensteps or something?

Reply:
It matches the example in the Eclipse plugins

Reply:
ah

Reply:
Not the Eclipse plugins, the examples haven't lived there for a year now.

Reply:
According to https://wpilib.screenstepslive.com/s/currentCS/m/java/l/145307-creating-your-benchtop-test-program, they do. This code also matches code on that ScreenSteps page.

Reply:
The examples are included with the Eclipse plugins, yes. They aren't maintained there any more — they live in the allwpilib repo now, so you should be basing any work off there.